src/PASopa_pack.sln

```diff
@@ -0,0 +1,36 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Why do we need separate solution file?

Reply:
this includes only the projects that need to be published as nuget packages, avoids running the pack job for each one

Reply:
let's see if we can avoid extra solution

Reply:
added an array parameter to the `pack` step to specify the projects to include